### ISSUE TITLE
Setting `shuffle=True` in deep learning featurization will break workflow prediction and evaluation

### ISSUE THREAD

**Issue by smcolby** [OpenADMET/openadmet_models#383]
Modifying the `shuffle` argument in deep learning featurizers (thereby configuring the resultant data loaders) will cause shuffled predictions. When then compared to `y_true`, the out-of-order result will be completely wrong. 

Ideally we should be able to choose whether to shuffle training data batches, but force no shuffling for validation and test. Currently there is only a single toggle to affect all splits.

For now, DO NOT SET `shuffle=True` per the issue with test set shuffling.
